lib/rspec/mocks/minitest_integration.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec/mocks'
+
+Minitest::Expectation.class_eval do
+  def to(*args, &block)
+    ::RSpec::Mocks::ExpectationTarget.new(target).to(*args, &block)
```

Comment:
RSpec's `expect(...).to`/`expect(...).not_to`/`expect(...).to_not` syntax has a bit of implementation trickiness around the fact that both rspec-mocks and rspec-expectations use that syntax, and both libraries must be usable individually (e.g. for a minitest user who uses one but not the other) and they also must be usable together.  Here's how this is currently implemented:

* Both libraries define an `ExpectationTarget` class (`RSpec::Expectations::ExpectationTarget` and `RSpec::Mocks::ExpectationTarget`).
* In rspec-mocks, the classes behind `receive`, `receive_messages`, `have_received` and `receive_message_chain` implement the rspec-expectations matcher protocol so that rspec-expectations' `expect` can treat them just like any other matcher without any knowledge of rspec-mocks.
* In rspec-mocks, we define `expect` in [such a way](https://github.com/rspec/rspec-mocks/blob/574e73b56eb703ae92a0b30be981961b98cb32ee/lib/rspec/mocks/example_methods.rb#L380-L386) that it is further up the superclass chain and would only get used if rspec-expectations is not included (either before or after rspec-mocks is included).

All that's to say that if the user is using rspec-mocks and rspec-expectations with minitest, `expect(...).to` is going to have to call `RSpec::Expectations::ExpectationTarget#to` instead of `RSpec::Mocks::ExpectationTarget#to` for things to work as the user expects.  This would perhaps need to delegate to the rspec-expectations `to` method here to work properly.  However, it's pretty clearly out of scope for this file to fix minitest's `expect` to work with rspec-expectations.  So I'm not exactly sure what we should do to make things work for all the cases....I really never expected minitest to add an `expect` method to interfere with ours.  I do think we should at least do some testing so that we understand how things work for the various cases (e.g. minitest with just rspec-mocks, minitest with just rspec-expectations, and minitest with both).  It would also be nice to confirm that whatever we have here does not interfere with rspec-expectations and minitest playing nicely together.

Does that all make sense?